java-repo-tools

```diff
@@ -0,0 +1 @@
+Subproject commit 74755a28f01aedf57992f473eb2d0201b1e04f2a
```

Comment:
@lesv This is the metadata that ties the submodule to the specific commit. Doing a git clone followed by a checkout of this commit number should be exactly the same. (This differs from subtrees, which don't require the extra clone, but then the histories were merged, which was pretty awful)